src/composition.cc

```diff
@@ -121,7 +125,7 @@ Composition::Ptr Composition::NewDecay(int delta) {
   if (atom_.size() == 0)
     return decayed;
 
-  decayed->atom_ = pyne::decayers::decay(atom_, 2419200.0 * delta);
+  decayed->atom_ = pyne::decayers::decay(atom_, static_cast<double>(secs_per_timestep) * delta);
```

Comment:
Should you also add a TODO here?

// TODO: when the backends get uint64_t support, the static_cast here should
+  // be removed.